identity_vc/src/macros.rs

```diff
@@ -0,0 +1,68 @@
+macro_rules! impl_builder_setter {
```

Comment:
I wonder if we should potentially move these macros over to core or maybe build a proc macro for builders in the derive crate.  Just a thought since if we want to follow the builder pattern then it may be a good idiom to adopt across all of the crates. There are also some battle tested builder derive crates already in the rust ecosystem we could look at to avoid building our own. `rust-derive-builder` and `typed_builder` come to mind.  Again, just a consideration, you implementation looks flexible enough.

Reply:
`rust-derive-builder` looks like a nice option - it supports *almost* everything **but** it doesn't have this special handling for `Vec<T>` (which ultimately becomes `OneOrMany<T>`). `typed_builder` is missing `TryInto` support, which is really nice to have.

I'll move to `identity_core` for now; maybe look at a fork of `rust-derive-builder` if our needs outgrow these macros (it needs a few changes anyways, eg. supporting `try_setter` with `strip_option`).

`#[derive(Builder)]` is definitely a **much** nicer pattern though.

Reply:
Alright, sounds like a good plan.  When you merge I will incorporate the pattern into the core for consistency.